string object_class
string id
string old_id
string origin
Point position
bool success
bool deleted